src/pages/settings/EditPasswordScreen.js

```diff
@@ -0,0 +1,149 @@
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import {ScrollView, View} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import _ from 'underscore';
+import Form from '../../components/Form';
+import Text from '../../components/Text';
+import TextInput from '../../components/TextInput';
+import {withCurrentUserPersonalDetailsDefaultProps} from '../../components/withCurrentUserPersonalDetails';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import * as User from '../../libs/actions/User';
+import compose from '../../libs/compose';
+import * as ErrorUtils from '../../libs/ErrorUtils';
+import * as ValidationUtils from '../../libs/ValidationUtils';
+import ONYXKEYS from '../../ONYXKEYS';
+import styles from '../../styles/styles';
+
+const propTypes = {
+    account: PropTypes.shape({
+        /** An error message to display to the user */
+        errors: PropTypes.objectOf(PropTypes.string),
+
+        /** Success message to display when necessary */
+        success: PropTypes.string,
+
+        /** Whether a sign on form is loading (being submitted) */
+        isLoading: PropTypes.bool,
+    }),
```

Comment:
This data now comes from the EDIT_PASSWORD_FORM key, no? DO we still need to subscribe to account?

Reply:
@luacmartins  I think we should keep account prop to show the error here: https://github.com/Expensify/App/blob/533e63ce8b13e573024784836cef0bb7e859e937/src/pages/settings/PasswordPage.js#L211

Reply:
Makes sense. I forgot that the API is still returning the errors to the account key.